### PR TITLE
update types for webextension-polyfill

### PR DESCRIPTION
**Pull request by Lusito**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

None of the above 2 checkboxes apply. Changes are automatically generated, except for manual fixes.

#### Some additional information if required:
webextension-polyfill is a polyfill for developing browser extensions. I use [webextension-polyfill-ts](https://github.com/Lusito/webextension-polyfill-ts) to generate types from official mozilla schema files located here:
- https://hg.mozilla.org/integration/autoland/raw-file/tip/toolkit/components/extensions/schemas/
- https://hg.mozilla.org/integration/autoland/raw-file/tip/browser/components/extensions/schemas/

So most of the changes you will see in PRs from me are introduced by changes in these schema files.
There are a couple of exceptions where I have to adjust the schema files, because the mozilla schema files are incorrect or incomplete:
- https://github.com/Lusito/webextension-polyfill-ts/tree/master/early-fixes
- https://github.com/Lusito/webextension-polyfill-ts/tree/master/fixes

### CODE REVIEW

File: types/webextension-polyfill/namespaces/tabs.d.ts [hunk 1]
```diff
@@ -700,7 +700,7 @@ export namespace Tabs {
          * The CookieStoreId used for the tab.
          * Optional.
          */
-        cookieStoreId?: string;
+        cookieStoreId?: string[] | string;
```
Comment (amcasey) [inline types/webextension-polyfill/namespaces/tabs.d.ts:703]: Confirm that this "Id" property might actually be multiple IDs?

**Review by amcasey — APPROVED** [review]

**Review by amcasey — CHANGES_REQUESTED** [review]
Wait, surely a version number should be bumped in a header somewhere?

File: types/webextension-polyfill/namespaces/tabs.d.ts (hunk 1 above)
Comment (Lusito) [reply types/webextension-polyfill/namespaces/tabs.d.ts:703]: Yes, this is a filter attribute, so you can enter either a string or a list of strings:  https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/query

**Comment by Lusito** [comment]
@amcasey There is no version bump since the library has not been updated. Since the polyfill only makes sure it works consistent on all browsers, the types don't have much to do with the polyfill itself.

The polyfill has not seen a new release in almost a year: https://www.npmjs.com/package/webextension-polyfill

**Review by amcasey — APPROVED** [review]
Thanks for following up!

**Comment by Lusito** [comment]
Ready to merge

**Comment by Lusito** [comment]
Thanks for the review!

**Pull request merged**